herddb-core/src/main/java/herddb/core/TableManager.java

```diff
@@ -2607,6 +2642,13 @@ private TableCheckpoint checkpoint(double dirtyThreshold, double fillThreshold,
             /* *** New pages handling *** */
             /* ************************** */
 
+            /*
+             * Preleva la pagina dirty "corrente", può essere tenuta in memoria perché durante un checkpoint
```

Comment:
please write in English language

Reply:
Oh! sorry, got carried away by documentation focus :D

Reply:
Done